metadata.yaml

```diff
@@ -33,7 +33,7 @@ requires:
     grafana-source:
         interface: grafana_datasource
     alertmanager:
-        interface: alertmanager
+        interface: alertmanager_dispatch
```

Comment:
It will be very helpful if you can add a commit message with the rationale underlying this choice of name.

Reply:
You mean in addition to the discussion in the linked issue #44?

Reply:
If you prefer you can just link to the issue. But a few lines copied and pasted  from there would also be be nice if you think git commit history is a valuable resource for future developers.

Reply:
Done:
https://github.com/canonical/alertmanager-operator/pull/14/commits/29514ca066294ef62cfcb5b28f58240a6d3fecf6